doc/src/manual/arrays.md

```diff
@@ -410,22 +410,43 @@ julia> sum(1/n^2 for n=1:1000)
 1.6439345666815615
 ```
 
-When writing a generator expression with multiple dimensions inside an argument list, parentheses
-are needed to separate the generator from subsequent arguments:
+When writing a generator expression in an argument list, parentheses can be omitted, provided the
+generator is the last argument:
 
-```julia-repl
-julia> map(tuple, 1/(i+j) for i=1:2, j=1:2, [1:4;])
-ERROR: syntax: invalid iteration specification
+```jldoctest
+julia> map(Tuple, i for i=1:4)
+4-element Vector{Tuple{Int64}}:
+ (1,)
+ (2,)
+ (3,)
+ (4,)
+
+julia> map(Tuple, (i,j) for i=1:2, j=3:4)
+2×2 Matrix{Tuple{Int64, Int64}}:
+ (1, 3)  (1, 4)
+ (2, 3)  (2, 4)
+
+julia> map(tuple, 1:4, i for i=5:8)
+4-element Vector{Tuple{Int64, Int64}}:
+ (1, 5)
+ (2, 6)
+ (3, 7)
+ (4, 8)
 ```
 
-All comma-separated expressions after `for` are interpreted as ranges. Adding parentheses lets
-us add a third argument to [`map`](@ref):
+If the generator is not the last in the argument list, then parentheses must be used to seperate it from
+subsequent arguments:
```

Comment:
This change is predicated on the first change removing the statement regarding what parentheses do here (separate expressions). It is less direct to state two important principles separately.

Reply:
This was added so the reason for needing parentheses will be seen.